include/manifold/manifold.h

```diff
@@ -422,6 +422,18 @@ class Manifold {
                          const std::vector<Smoothness>& sharpenedEdges = {});
   ///@}
 
+  /** @name Fillet
+   * Create a rounding corner
+   */
+  ///@{
+  static Manifold Fillet(const MeshGL&, double radius,
+                         const std::vector<size_t>& selectedEdges = {});
```

Comment:
So, the API here is tricky - I can see why you're basing this on MeshGL input, since that allows you to select edges, and that's what I did for one of the Smoothing constructors. However, I think I'd prefer to avoid edge-selection APIs as much as we can. I think I'd prefer a Fillet command that fillets all edges beyond a threshold angle, and then it can be non-static. Do you think that's doable?

Reply:
I think the fillet in pure geometric way won't be that robustness, See corner case 1. It heavily rely on the request radius and adjacent relationship. If so, the result might be out of control.
But I think it would be extremely valuable if there is a full robust fillet method, and i'll keep looking for it.

Reply:
Yeah, Manifold's core ethos is robustness - we want find an algorithm that doesn't have edges cases, rather than trying to fix edges cases one by one. In computational geometry, edge cases tend to be the norm, rather than the exception.